### DISCUSSION TITLE
Extending User profile views natively with DataViews

### DISCUSSION THREAD

**Discussion by flexseth** [WordPress/developer-blog-content#298] [Topic Ideas]
Use case: I want to add "armor" to a user profile, to match MMORPG games like World of Warcraft.

Example: Add "equipment" custom post type that has armor class, affects, and also allowing these pieces of armor to be attached to a user profile. 

"worn on body"
"worn on arms"
"worn on legs" 

etc..

In an ideal situation, you could search for equipment, and attach it to a user profile via a DataView.

Drag and drop functionality would be highly desirable, but a simple POC works, too.

Has many other use cases!

cc: @justintadlock

**Comment by bph** [comment]
This topic was approved in the 9/5 editorial group meeting. Please continue the conversation on the issue
- #305
